js/main.js

```diff
@@ -118,13 +118,19 @@ function viewSwapping(dataView) {
   if (dataView === 'profile') {
     $views[1].innerHTML = '';
     $views[1].appendChild(profileRender(data));
-  } else {
+  } else if (dataView === 'edit-profile') {
     $form.elements.avatarUrl.value = data.profile.avatarUrl;
     $form.elements.username.value = data.profile.username;
     $form.elements.fullName.value = data.profile.fullName;
     $form.elements.location.value = data.profile.location;
     $form.elements.bio.value = data.profile.bio;
     $image.setAttribute('src', $form.elements.avatarUrl.value);
+  } else if (dataView === 'entries') {
+    var $ol = document.querySelector('ol');
+    $ol.innerHTML = '';
+    for (var i = 0; i < data.entries.length; i++) {
```

Comment:
You should be doing this loop when the document first loads to prepare for any pre-existing entries 👍